public/locales/zhtw/common.json

```
@@ -0,0 +1,11 @@
+{
+  "title": "讓你用最輕鬆的方式產生",
+  "readme": "Readme",
```

Can you make this `README` to conform with the rest of the languages
```suggestion
  "readme": "README",
```